### ISSUE TITLE
Does this support any anit-cheats? 

### ISSUE THREAD

**Issue by nitrogen69** [Jire/Charlatano#630]
People constantly ask if the cheat is safe to use on FaceIT, ESEA and so on.
So im asking the question we all wanna have the answer to.

Does Charlatano 1.0.8 work with EAC, FACEIT, ESEA and so on? 💯

**Comment by JimboSitch** [comment]
Yes it does say

**Comment by Plasma2450** [comment]
Nope, these anti-cheats are slightly more sophisticated and can't see any usermode cheats being undetected let along open source ones.
